exercises/Graphing-3-Python.md

```diff
@@ -10,16 +11,16 @@ look at the problem empirically for mammals.
 
 Download some
 [mammal life history data](http://esapubs.org/archive/ecol/E084/093/Mammal_lifehistories_v2.txt)
-from the web. You can do this either directly in the program using urllib or
+from the web. You can do this either directly in the program using `urllib` or
 download the file to your computer using your browser and import it from there.
 
 Import the data into a Pandas data frame. There are some extra blank lines at
 the end of this file, so get rid of them by using the optional `read_csv()`
-argument, `skip_footer=7`.
+argument, `skip_footer = 7`.
```

Comment:
> no spacing around =.

Done. Not sure why `AnyStr = None` has a space in the function call in the
alternate example

def munge(sep: AnyStr = None):


On Mon, Jul 20, 2015 at 5:35 AM, Ethan White <notifications@github.com>
wrote:

> In exercises/Graphing-3-Python.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/150#discussion_r34982977>
> :
>
> >  download the file to your computer using your browser and import it from there.
> >
> >  Import the data into a Pandas data frame. There are some extra blank lines at
> >  the end of this file, so get rid of them by using the optional `read_csv()`
> > -argument, `skip_footer=7`.
> > +argument, `skip_footer = 7`.
>
> Since this is a function argument there should be no spacing around =.
> See https://www.python.org/dev/peps/pep-0008/#other-recommendations
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/150/files#r34982977>
> .
>